kvm/src/ioctl_wrap.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: Alex suggested in your initial PR to rename this to ioctl.rs. I also think it sounds better that way.

Reply:
I cannot rename it to ioctl as it would conflict with the ioctl defined in sys_util. Any other suggesstion?
`
   |                                                                            
20 | mod ioctl;                                                                 
   | ---------- previous definition of the module `ioctl` here                  
...                                                                             
31 | use sys_util::{ioctl, ioctl_with_ref, ioctl_with_val};                     
   |                ^^^^^ `ioctl` reimported here                               
   |                                                   `